TicketDesk/ngWebClient-StandAlone/src/app/ticket-list/ticket-list.component.ts

```diff
@@ -18,10 +18,9 @@ export class TicketListComponent implements OnInit {
   currentPage: number;
   ngOnInit() {
     this.displayList = [];
-    for (const ticket of this.ticketListResults.ticketList) {
-      if (ticket) { // ticket is of type ticket
+    // filter removes objects not of type ticket or null/undefined
+    for (const ticket of this.ticketListResults.ticketList.filter( x => x)) {
         this.displayList.push({'ticket': ticket, 'checked': false});
-      }
```

Comment:
this.displayList = this.ticketListResults.ticketList
.filter( x => x)
.map( ticket => {'ticket': ticket, 'checked':false});

so it WOULD be something like this but let's NOT bikeshed it lol